src/Microsoft.AspNet.OData.Shared/EdmODataAPIHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.AspNet.OData
+{
+
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update.
+    /// This is the handler for data modification where there is no CLR type.
+    /// </summary>
+    internal abstract class EdmODataAPIHandler
```

Comment:
Why can't this be an `interface` instead of `abstract class`?

Reply:
To maintain pattern similar to internal abstract class ODataAPIHandler<TStructuralType>: IODataAPIHandler where TStructuralType : class